garage/envs/noisy_env.py

```diff
@@ -22,7 +21,7 @@ def __init__(
         Serializable.quick_init(self, locals())
         super(NoisyObservationEnv, self).__init__(env)
         self.obs_noise = obs_noise
-        self._action_flat_dim = flat_dim(self.action_space)
+        self._action_flat_dim = self.action_space.low.shape
```

Comment:
this is a shape, not a flat_dim. `np.prod(self.action_space.low.shape)`

I might support keeping flat_dim just to stop people from getting this wrong.

Reply:
I'm not sure how I missed np.prod(). My b